app/src/main/java/com/github/gotify/messages/MessagesActivity.java

```diff
@@ -239,6 +241,16 @@ protected void onUpdateApps(List<Application> applications) {
                     .placeholder(R.drawable.ic_placeholder)
                     .resize(100, 100)
                     .into(t);
+
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                NotificationSupport.createChannel(
```

Comment:
This should be done inside the WebSocketService, as the MessageActivity doesn't need to be executed for stating the WebSocketService, like when booting the phone -> com.github.gotify.init.BootCompletedReceiver

Reply:
But how would i get a list of application names? I tried finding occurences of ApplicationHolder.get(), it only shows up there (and in the share activity)

Reply:
Actually, i found out we dont really need to have an applist anyway. We can create the channel by ID alone, and then update the channelname later. This way we can even skip the fallback, because we can create channels on demand. The only downside is that the channel name is not correct until it updates eventually, but that seems reasonable